### PR TITLE
Add table for logging telerivet events

### PR DESCRIPTION
**Pull request by gcapalbo**

### CODE REVIEW

File: corehq/apps/telerivet/tasks.py [hunk 1]
```diff
@@ -16,6 +17,28 @@
...
+            time_created=kwargs["time_created"],
+            time_sent=kwargs["time_sent"],
+            contact_id=kwargs["contact_id"],
+            phone_id=kwargs["phone_id"],
+            service_id=kwargs["service_id"],
+            project_id=kwargs["project_id"],
+            secret=kwargs["secret"],
+        )
+        log.save()
+    except Exception as e:
```
Comment (dimagimon) [inline corehq/apps/telerivet/tasks.py]: F841 local variable 'e' is assigned to but never used

File: corehq/apps/telerivet/tasks.py [hunk 2]
```diff
@@ -16,6 +17,28 @@
...
+            from_number=kwargs["from_number"],
+            from_number_e164=kwargs["from_number_e164"],
+            to_number=kwargs["to_number"],
+            time_created=kwargs["time_created"],
+            time_sent=kwargs["time_sent"],
+            contact_id=kwargs["contact_id"],
+            phone_id=kwargs["phone_id"],
+            service_id=kwargs["service_id"],
+            project_id=kwargs["project_id"],
+            secret=kwargs["secret"],
```
Comment (czue) [inline corehq/apps/telerivet/tasks.py]: is there a reason you didn't just do `IncomingRequest(**kwargs)`?

Comment (czue) [inline corehq/apps/telerivet/tasks.py]: maybe there is extra stuff in there? seems like you could clean this up a little given how repetitive this is.

File: corehq/apps/telerivet/views.py [hunk 3]
```diff
@@ -7,18 +7,20 @@
...
+        "from_number": request.POST.get("from_number"),
+        "from_number_e164": request.POST.get("from_number_e164"),
+        "to_number": request.POST.get("to_number"),
+        "time_created": request.POST.get("time_created"),
+        "time_sent": request.POST.get("time_sent"),
+        "contact_id": request.POST.get("contact_id"),
+        "phone_id": request.POST.get("phone_id"),
+        "service_id": request.POST.get("service_id"),
+        "project_id": request.POST.get("project_id"),
+        "secret": request.POST.get("secret"),
```
Comment (czue) [inline corehq/apps/telerivet/views.py]: (similar comment here)

**Comment by czue** [comment]
don't think comments need to block merge, but curious to get your quick input first

File: corehq/apps/telerivet/tasks.py [hunk 4]
```diff
@@ -16,6 +17,15 @@
 
 @task(queue=CELERY_QUEUE)
 def process_incoming_message(*args, **kwargs):
+    try:
+        from corehq.apps.telerivet.views import TELERIVET_INBOUND_FIELD_MAP
+        fields = {a: kwargs[a] for (a, b) in TELERIVET_INBOUND_FIELD_MAP}
+        log = IncomingRequest(**fields)
+        log.save()
+    except Exception as e:
```
Comment (dimagimon) [inline corehq/apps/telerivet/tasks.py]: F841 local variable 'e' is assigned to but never used

**Comment by gcapalbo** [comment]
@czue I thought about passing in kwargs when instantiating the IncomingRequest, but that seemed fragile to me as someone could easily add another parameter to the task and it would break.  I just updated it to address the repetitiveness.

**Pull request merged**